src/main/java/teammates/common/util/Const.java

```diff
@@ -1354,6 +1357,15 @@ private Const() {
         public static final String FEEDBACK_SESSION_ADDED =
                 "The feedback session has been added. "
                 + "Click the \"Add New Question\" button below to begin adding questions for the feedback session.";
+        public static final String FEEDBACK_SESSION_ADDED_OWN_QUESTIONS =
+                "Feedback session using your own questions has been added. "
+                + "Click the \"Add New Question\" button below to begin adding questions for the session.";
+        public static final String FEEDBACK_SESSION_ADDED_TEAM =
+                "Feedback session using team evaluation template has been added. "
+                + "Click the \"Add New Question\" button below to begin adding questions for the session.";
```

Comment:
Can extract this sentence out since it's repeated multiple times? Also, standardise it with the one in `FEEDBACK_SESSION_ADDED`.